ConnectorRevit/RevitSharedResources/Models/TransactionManager.cs

```diff
@@ -186,7 +186,7 @@ public static TResult ExecuteInTemporaryTransaction<TResult>(Func<TResult> funct
         t.Start();
         result = function();
       }
-      catch
+      catch (Autodesk.Revit.Exceptions.ApplicationException)
       {
         // ignore because we're just going to rollback
```

Comment:
But what happened, don't we need something to appear in the report?

Reply:
I think Connor makes a valid point. But ideally we'd want to know if something went wrong in a sub-transaction.

I wouldn't change the existing behaviour right now, but given this is in the generic `TransactionManager`, having a way to know if a temp transaction went wrong (and what went wrong exactly) seems useful.

Maybe the solution here would be to capture the exception, rollback the transaction in a `finally` catch and throw right after (though this also sounds admittedly hacky)

Reply:
I think we should open a ticket to refactor this in a way that it could throw, but right now just adding it to the logs should be useful enough for us.